### DISCUSSION TITLE
Indicator on the chart

### DISCUSSION THREAD

**Discussion by maxah** [quantform/quantform#70] [Q&A] [answered]
What is the indicator on the chart on the home page? How does it work?

**Comment by Majchrzak** [comment, marked as answer]
If I recall correctly, it's some variation of Hurst indicator. 

This one on homepage is based on HCCC_LB indicator created by LazyBear, you can find one on TradingView.

**Reply by maxah** [reply]
Thanks, I'll take a look.
